src/payment/data/sagas.test.js

```diff
@@ -863,6 +872,225 @@ describe('saga tests', () => {
     });
   });
 
+  describe('handlePaymentState', () => {
```

Comment:
These tests test the Saga, and in a couple the reducer as well with some living alterable state.